### PR TITLE
SSO load tests

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: tests/locust/test_sso.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        self.client.get(get_url('sso:password_set'))
+
+    @task
+    def password_reset(self):
+        self.client.get(get_url('sso:password_reset'))
+
+    @task
+    def email(self):
+        self.client.get(get_url('sso:email'))
+
```
Comment (monikasulik) [inline tests/locust/test_sso.py]: hmmm, interesting that flake8 didn't pick up the 3 lines instead of 2

File: tests/api/test_connection.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
-from tests.settings import DIRECTORY_API_URL
+from tests import get_absolute_url
 
 
 def test_api_connection():
-    response = requests.get('{}/docs/'.format(DIRECTORY_API_URL))
-    assert response.ok
+    response = requests.get(get_absolute_url('api:docs'))
+    assert response.status_code == httplib.FORBIDDEN
```
Comment (monikasulik) [inline tests/api/test_connection.py]: Why has this changed from ok to forbidden? Or was the original assert wrong?

File: makefile [hunk 3]
```diff
@@ -59,7 +82,6 @@ DOCKER_SET_DIRECTORY_UI_ENV_VARS := \
 DOCKER_SET_DIRECTORY_TESTS_ENV_VARS := \
 	export DIRECTORY_TESTS_DIRECTORY_API_URL=http://directory_api_webserver:8000; \
 	export DIRECTORY_TESTS_DIRECTORY_SSO_URL=http://www.sso.dev.playground.directory.uktrade.io/; \
-	export DIRECTORY_TESTS_DIRECTORY_UI_LOAD_URL=http://www.dev.playground.directory.uktrade.io; \
 	export DIRECTORY_TESTS_DIRECTORY_UI_URL=http://directory_ui_webserver:8001; \
```
Comment (monikasulik) [inline makefile]: Shouldn't this be changed to dev playground then? Because I don't think we actually want to load test docker on each PR, right?

Comment (richtier) [inline makefile]: strictly, yes. This PR can live without that change though. Its handled in [this other PR](https://github.com/uktrade/directory-tests/pull/4)

**Pull request merged**